common/src/main/java/org/apache/falcon/persistence/PersistenceConstants.java

```diff
@@ -67,4 +67,7 @@ private PersistenceConstants(){
     public static final String DELETE_BACKLOG_METRIC_INSTANCE = "DELETE_BACKLOG_METRIC_INSTANCE";
     public static final String GET_ALL_BACKLOG_INSTANCES = "GET_ALL_BACKLOG_INSTANCES";
     public static final String DELETE_ALL_BACKLOG_ENTITY_INSTANCES ="DELETE_ALL_BACKLOG_ENTITY_INSTANCES";
+    public static final String GET_ALL_RECIPES = "GET_ALL_RECIPES";
+    public static final String GET_RECIPE_LOCATION = "GET_RECIPE_LOCATION";
+    public static final String DELETE_ALL_RECIPES = "DELETE_ALL_RECIPES";
```

Comment:
DELETE A specific recipe?
and also can you use extensions as the suffix/prefix rather than recipe?